webapp/javascript/components/FileUploader.tsx

```diff
@@ -47,8 +53,7 @@ export default function FileUploader({ onUpload }: Props) {
   });
 
   const onRemove = () => {
-    setFile(null);
-    onUpload(null);
+    setFile(null, null);
```

Comment:
I am, personally, not a fan of `null`s. I would much rather use `Optional`s. But this is one of the cases it's fine IMO, up to you.

Reply:
Using optionals properly would require to also handle optional flamebearer I think, maybe something that can be considerend independently from this PR?